test/preprocessing/vtk2trixi/vtk2trixi.jl

```diff
@@ -0,0 +1,81 @@
+@testset verbose=true "`vtk2trixi`" begin
+    output_directory = joinpath("test/preprocessing/readvtk")
```

Comment:
```suggestion
    output_directory = joinpath("test" , "preprocessing", "readvtk")
```
Please also adapt for the following `joinpath()` calls

Reply:
```suggestion
    mktempdir() do tmpdir
        # testsets 
    
    end
```
Or even better, use `mktempdir()`. Then the temporary directory is automatically deleted when the process exits.